features/step_definitions/networking.rb

```diff
@@ -305,6 +305,15 @@
   end
 end
 
+Given /^the AllowNamespaceAndPod policy is applied to the "(.+?)" namespace$/ do | project_name |
+  ensure_admin_tagged
+
+  @result = admin.cli_exec(:create, n: project_name , f: "#{BushSlicer::HOME}/testdata/networking/networkpolicy/allow-ns-and-pod.yaml")
```

Comment:
@asood-rh I think we prefer to use the generic step to download get the testdata first using ```step %Q(I obtain test data file "networking/networkpolicy/allow-ns-and-pod.yaml")``` and then you can use `f: "allow-ns-and-pod.yaml"`

Reply:
@pruan-rht What is the advantage of using the generic step to download the data file before applying it when the current implementation does both operations in on step?
I am following the pattern used for apply another policy.

Reply:
+1 The idea is that these files may be moved or reorganized and things can remain transparent to users.

Reply:
@akostadinov yes, it is my first PR.